doc/users/next_whats_new/callables_for_formatting_sankey_labels.rst

```diff
@@ -0,0 +1,46 @@
+Support callable for formatting of Sankey labels
+------------------------------------------------
+
+The `format` parameter of `matplotlib.sankey.Sankey` can now accept callables.
+
+This allows the use of an arbitrary function to label flows, for example allowing
+the mapping of numbers to emoji.
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    from matplotlib.sankey import Sankey
+    import math
+
+
+    def display_in_cats(values, min_cats, max_cats):
+        def display_in_cat_scale(value):
+            max_value = max(values, key=abs)
+            number_cats_to_show = \
+                max(min_cats, math.floor(abs(value) / max_value * max_cats))
+            return str(number_cats_to_show * '🐱')
```

Comment:
At least for me the way GH renders this I don't see the cat emoji, but it is in there.  This renders as https://51382-1385122-gh.circle-artifacts.com/0/doc/build/html/users/next_whats_new/callables_for_formatting_sankey_labels.html 
![image](https://user-images.githubusercontent.com/199813/103567581-120d8600-4e92-11eb-88b8-36f95acad13c.png)